### PR TITLE
Fixed display of last items in stock

### PR DESCRIPTION
**Pull request by thierrymarianne**
| Questions | Answers |
| --- | --- |
| Branch? | develop |
| Description? | When there is less items of a product than what has been defined in Product Settings, we display a warning message to customers |
| Type? | bug fix |
| Category? | CO |
| BC breaks? | no |
| Deprecations? | no |
| Fixed ticket? | http://forge.prestashop.com/browse/BOOM-383 |
| How to test? | Set threshold in back-office (Shop Parameters > Product Settings), update the quantity of a product in stock, check if product page in front-office has warning message |

### CODE REVIEW

File: themes/classic/_dev/css/partials/_variables.scss [hunk 1]
```diff
@@ -8,6 +8,7 @@ $brand-secondary: #f39d72;
 $brand-success: #4cbb6c;
 $brand-warning: #ff9a52;
 $brand-danger: #ff4c4c;
+$last-items: #ff9a52;
```
Comment (julienbourdeau) [inline themes/classic/_dev/css/partials/_variables.scss]: @nihco2 What do you think of this variable? Maybe we should use the brand warning color for consistancy?

**Comment by vincentbz** [comment]
Tested and ok, thanks !

File: themes/classic/_dev/css/partials/_variables.scss (hunk 1 above)
Comment (nihco2) [inline themes/classic/_dev/css/partials/_variables.scss]: @julienbourdeau @thierrymarianne Yes, please don't duplicate variable if it's the same color as $brand-warning

**Comment by mickaelandrieu** [comment]
@thierrymarianne this need a rebase here :)

**Review by maximebiloe — APPROVED** [review]

**Comment by maximebiloe** [comment]
Thank you

**Pull request merged**